### DISCUSSION TITLE
No replay after opened a ticket

### DISCUSSION THREAD

**Discussion by MosesPond** [vercel/community#4242] [Help]
### Summary

I opened a support ticket on the 10th of Septmber and haven't gotten any replay yet.
any other way to contact vercel ?i

### Steps to Reproduce

we use vercel to upload our website, i had some issues with reading comments of teamates on the website, so i opened a ticket but i haven't got any replay.

**Comment by amyegan** [comment]
Hi @MosesPond. Sorry you've been waiting so long. If you let me know the ticket number, I can check on it for you.

You can find more info about response times and how Vercel support works at https://vercel.com/guides/vercel-support-queue-time
> Tickets are processed by the order in which they are received. This means that, when you submit your request, you will need to wait for all requests submitted by customers on the same plan as you to be fulfilled before a response is received.

**Reply by MosesPond** [reply]
#00157082
the comments sometimes have issues of not showing. i did found after few days the feature to view them all but still i noticed the vercel js throws exceptions on devleoper tools console regarding the team comments.
